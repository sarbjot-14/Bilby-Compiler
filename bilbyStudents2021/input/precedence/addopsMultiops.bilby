main{
	print 3 + 5 + 7$n;
	print 35 - 5 + 7$n;
	print 3 - 5 - 7$n;
	print 3 + 5 * 4$n;
	print 5 * 4 +3  $n;
	print -3 +3+ 8 / 4 -4 *2   $n;
	
	
}
