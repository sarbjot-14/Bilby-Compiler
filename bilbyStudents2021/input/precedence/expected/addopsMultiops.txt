15
37
-9
23
23
-6
